ansible/roles/java-base/tasks/main.yml

```diff
@@ -57,6 +57,7 @@
   check_mode: no
   register: adoptopenjdk_metadata
   uri:
+    body: ""
     body_format: json
```

Comment:
Is this necessary? It may be the cause of the null value being sent by default

Reply:
I'll check what happens without it.

Reply:
Looks like it is isn't needed. I've removed both `body` and `body_format` options.